src/main/tables.ts

```diff
@@ -52,6 +52,9 @@ export async function loadTables(): Promise<ImportedTable[]> {
       ret.find(({ name }) => name == row.tablename)?.aidColumns.push(row.objname.split('.').at(-1)),
     );
 
+    const tables = ret.map((table) => `'${table.name}'`).join(', ');
```

Comment:
how about we drop the `'`s to limit the amount of quotation in the logs?

Actually, I see that our logs might be inconsistent in that aspect, within these two files. I think removing all quotation from all logs would work best, but not sure.

Reply:
I think it helps clarify if any special characters appear in the name (like spaces or commas or dots).